app/webpacker/css/admin_v3/components/pagination.scss

```diff
@@ -39,10 +39,16 @@
       cursor: pointer;
     }
 
-    &:hover:not(.disabled) {
+    &:hover:not(.disabled):not(.gap) {
```

Comment:
Building upon my previous comment, how about we just scope it to clickable links? 
```suggestion
    &[href] {
```

Reply:
Yes! ad3f9001c3